science/proxmark3-iceman/Portfile

```diff
@@ -33,14 +33,35 @@ depends_lib-append  port:readline \
 
 depends_build-append \
                     port:arm-none-eabi-gcc \
-                    port:arm-none-eabi-binutils
+                    port:arm-none-eabi-binutils \
+                    port:coreutils
 
+# although the upstream repository does not have a ./configure script,
+# the buildsystem itself does rely on things usually found in ./configure
+# like for instance, pkg-config.
+#
+# this is why i (@linuxgemini) has decided to set variables related to
+# configure phase instead of making new ones before passing them
+# to build and destroot envs.
+
+set python_prefix   ${frameworks_dir}/Python.framework/Versions/3.9
+configure.python    ${prefix}/bin/python3.9
```

Comment:
default Python version in MacPorts is 3.11 and that should be used if possible

Reply:
Let me check if upstream has any issues about it, will change then

Reply:
changed to 3.11